### ISSUE TITLE
Docs: improve ruler and alertmanager configuration doc

### ISSUE THREAD

**Issue by pracucci** [grafana/mimir#1786]
We could improve ruler and alertmanager documentation to explain:
- What is a recording and alerting rule? Just a sentence to set the basis
- How they're related and required wiring (e.g. ruler sends events to alertmanager, then alertmanager sends notifications to configured receivers)
- How to configure rules in the ruler (https://grafana.com/docs/mimir/latest/operators-guide/architecture/components/ruler/ doesn't even mention `mimirtool` or Grafana Alerting UI)
- How to configure alertmanager config (https://grafana.com/docs/mimir/latest/operators-guide/architecture/components/alertmanager/ doesn't mention Grafana Alerting UI)

**Comment by kotchaosu** [comment]
Would it work in the similar way as in Cortex? For Cortex I'm creating Cortex and Cortex-Alertmanager datasources which allow me to configure the alerting, recording rules and notification points in Grafana UI. Tried to do the same with Mimir, but for Mimir Alertmanager the datasource is invalid due to missing required endpoints. From Mimir Nginx:

```
2022/05/25 12:42:23 [error] 9#9: *46 open() "/etc/nginx/html/api/v2/status" failed (2: No such file or directory), client: 10.233.86.239, server: , request: "GET /api/v2/status HTTP/1.1", host: "mimir-mimir-poc-nginx"
10.233.86.239 - - [25/May/2022:12:42:23 +0000]  404 "GET /api/v2/status HTTP/1.1" 154 "-" "Grafana/8.5.0" "10.10.10.126, 10.233.86.221, 10.233.86.221"
10.233.86.239 - - [25/May/2022:12:42:23 +0000]  404 "GET /alertmanager/api/v2/status HTTP/1.1" 35 "-" "Grafana/8.5.0" "10.10.10.126, 10.233.86.221, 10.233.86.221"
```

**Comment by pracucci** [comment]
> Would it work in the similar way as in Cortex? For Cortex I'm creating Cortex and Cortex-Alertmanager datasources which allow me to configure the alerting, recording rules and notification points in Grafana UI.

Yes, it works the same way. We also show it in this Mimir tutorial:
https://grafana.com/tutorials/play-with-grafana-mimir/

> Tried to do the same with Mimir, but for Mimir Alertmanager the datasource is invalid due to missing required endpoints.

Looks like a configuration issue. Can you open a discussion at https://github.com/grafana/mimir/discussions sharing more information about your setup, please?

**Comment by kotchaosu** [comment]
Yeah, sure - thanks a lot for quick response!
